src/charm.py

```diff
@@ -17,12 +20,51 @@ class GraylogCharm(CharmBase):
     def __init__(self, *args):
         super().__init__(*args)
         self.framework.observe(self.on.config_changed, self._on_config_changed)
+        self.framework.observe(self.on.stop, self._on_stop)
 
     def _on_config_changed(self, _):
-        current = self.model.config["thing"]
-        if current not in self._stored.things:
-            logger.debug("found a new thing: %r", current)
-            self._stored.things.append(current)
+        self._configure_pod()
+
+    def _on_stop(self, event):
+        self.unit.status = MaintenanceStatus('Pod is terminating')
```

Comment:
@justinmclark what do you think about having pre-defined messages somewhere so we always send **exactly** the same message for the same situation?

For example, in the same method `_on_stop` in [elasticsearch-operator](https://github.com/canonical/elasticsearch-operator/blob/master/src/charm.py#L32) we are using a slightly different message. (note the ".")

I know it is a silly thing, but may be it is something that can be standardized.

Reply:
I think that's a good idea and we should if there is a good way to standardize messaging between charms. My first thought would be to have it somewhere in Operator Framework and then we can just import the message:

e.g.
```python
from ops.model import StoppingStatusK8s
```

Maybe we raise this with the team?